tests/assets/sign_detached.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+FILE_PATH=$1
+SIGNATURE_PATH="$1.asc"
+: "${PULP_SIGNING_KEY_FINGERPRINT:="Pulp QE"}"
+
+# Create a detached signature
+gpg --quiet --batch --homedir ~/.gnupg/ --detach-sign --default-key "${PULP_SIGNING_KEY_FINGERPRINT}" \
```

Comment:
Yes, yes, yes!!!
Can you maybe take a tour and update the docs for signing services everywhere to use that style?